dm/worker/server.go

```diff
@@ -135,20 +136,20 @@ func (s *Server) Close() {
 
 // StartSubTask implements WorkerServer.StartSubTask
 func (s *Server) StartSubTask(ctx context.Context, req *pb.StartSubTaskRequest) (*pb.OperateSubTaskResponse, error) {
-	log.Infof("[server] receive StartSubTask request %+v", req)
+	log.L().Info("", zap.String("request", "start subtask"), zap.Stringer("payload", req))
```

Comment:
do we need to unify `request` with _server.go_ in DM-master? (`start subtask` or `StartSubTask`).

Reply:
good suggestion